app/views/shared/_rollbar.html.erb

```diff
@@ -34,14 +34,30 @@
   var _rollbarConfig = {
     accessToken: "a79df03b6e734ef78f9149de2329d886",
     captureUncaught: true,
+    captureUnhandledRejections: true,
     transform: transform,
-    checkIgnore: checkIgnore
+    checkIgnore: checkIgnore,
+    autoInstrument: {
```

Comment:
@alexsoble I've verified in the demo site that this looks right, but can you double-check and 👍 that this seems correct?